### ISSUE TITLE
Visualization of results

### ISSUE THREAD

**Issue by RestInTheEnd** [morancyc/USNet#4]
How to get the uncertainty map？

**Comment by morancyc** [comment]
The uncertainty map can be obtained by `u = args.num_classes / s`, you can insert this code after line 29 in test.py.

**Comment by MalekWahidi** [comment]
How do i visualize 'u' like in the GIF?

**Comment by morancyc** [comment]
> How do i visualize 'u' like in the GIF?

To visualize 'u' overlapping on original rgb image, you can refer to this code (insert this code after line 45 in test.py):

            u = args.num_classes / s

            u = u.view(args.crop_height, args.crop_width)
            u = u.detach().cpu().numpy()
            u = np.array(u)

            # make save folder
            save_u_folder = './result/u_log'
            save_map_folder = './result/u_map'
            if not os.path.exists(save_u_folder):
                os.makedirs(save_u_folder)
                os.makedirs(save_map_folder)
            save_u_path = os.path.join(save_u_folder, save_name)
            save_map_path = os.path.join(save_map_folder, save_name)

            # Visualize uncertainty map with log function
            plt.figure()
            plt.set_cmap('jet')
            plt.axis('off')
            plt.imshow(np.log(u))
            plt.savefig(save_u_path, dpi=720, bbox_inches='tight', pad_inches=0)

            # read uncertainty map and rgb image, and overlay the two images
            u = cv2.imread(save_u_path)
            u = cv2.resize(u, (args.crop_width, args.crop_height))
            im = cv2.imread(sample['img_path'][0])
            im = cv2.resize(im, (args.crop_width, args.crop_height))
            u_map = cv2.addWeighted(u, 0.6, im, 0.4, 0)
            cv2.imwrite(save_map_path, u_map)

Please note that, we apply log function on 'u' for clear visualization.
